Src/Support/GoogleApis.Core/Apis/Http/ConfigurableMessageHandler.cs

```diff
@@ -173,6 +174,19 @@ public void RemoveExecuteInterceptor(IHttpExecuteInterceptor interceptor)
 
         #endregion
 
+        private int _loggingRequestId = 0;
+
+        private ILogger _instanceLogger = Logger;
+
+        /// <summary>
+        /// For testing only

```

Comment:
Can I ask when it is acceptable  to use _ in front of a variable?  Thought this was a no no.